CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
+## [v1]
```

Comment:
Can we go `release-1`? I know we haven't, but if we are okay with it, we should change it now?